nrf5/boards/nrf51822_ac_s110.ld

```diff
@@ -9,7 +9,7 @@ MEMORY
     FLASH (rx)      : ORIGIN = 0x00000000, LENGTH = 0x040000 /* entire flash, 256 KiB */
     FLASH_ISR (rx)  : ORIGIN = 0x00018000, LENGTH = 0x000400 /* sector 0, 1 KiB */
     FLASH_TEXT (rx) : ORIGIN = 0x00018400, LENGTH = 0x027c00 /* 159 KiB */
-    RAM (xrw)       : ORIGIN = 0x20002000, LENGTH = 0x006000 /* 24 KiB */
+    RAM (xrw)       : ORIGIN = 0x20001800, LENGTH = 0x002800 /* 12 KiB */
```

Comment:
Hi @glennrub,

first of all thank you so much for taking your time to comment on the code. I will try to contribute as much as I can to speed up the development of this branch. But please bear with me, I'm still a novice :)

Ok, I have core51822 board which is an AA variant with 16k RAM. changing the address the starting address was to see if I can get more ram available for the application. 
I will revert back the code to the original state and use the AA linker.